tablo/models.py

```diff
@@ -432,25 +451,32 @@ def _validate_where_clause(self, where):
         parsed = self._parse_where_clause(where)
 
         if parsed is None:
-            return True
+            return
         elif parsed[1]:
-            return False  # More than one statement means SQL injection.
-        else:
-            return self._validate_fields(parsed[0])
+            raise SQLInjectionError('Invalid where clause')
```

Comment:
This is raised when there is more than one SQL statement passed in. It means there was a semi-colon following the `WHERE` with more SQL following it, which is unlikely to be a mistyped field. Do you think it needs a comment?

Below we validate the fields, which may also include SQL injection, but our measure for validating them is whether or not they exist in the queried tables. So `_validate_fields` raises `InvalidFieldsError` instead of assuming intent.